### PR TITLE
Tweaks to security credits lists

### PR DESCRIPTION
**Pull request by ushkarev**

### CODE REVIEW

**Comment by ushkarev** [comment]
@maxf this one's ready for you now :)

File: mtp_noms_ops/assets-src/javascripts/modules/security-forms.js [hunk 1]
```diff
@@ -56,6 +56,9 @@ exports.SecurityForms = {
 
         $creditList.find('caption').remove();
         $creditList.find('.CollapsingTable').removeClass('CollapsingTable');
+        $creditList.find('tr th:first-of-type, tr td:first-of-type').each(function() {
+          $(this).remove();
+        });
         $thisRow.find('td').each(function () {
```
Comment (maxf) [inline mtp_noms_ops/assets-src/javascripts/modules/security-forms.js]: Does this work in IE8? first-of-type isn't supported

Comment (ushkarev) [inline mtp_noms_ops/assets-src/javascripts/modules/security-forms.js]: i figured jquery would deal with that, it doesn't use native selector mechanisms does it?

Comment (maxf) [inline mtp_noms_ops/assets-src/javascripts/modules/security-forms.js]: We should test anyway

**Pull request merged**